cf/query.py

```diff
@@ -207,6 +207,7 @@ def __init__(
         exact=True,
         rtol=None,
         atol=None,
+        **kwargs,
```

Comment:
I think that it's better to have two new keyword arguments Boolean `open_lower` and `open_lower` (or perhaps one new argument `open` that is a tuple of 2 bools?) than to use unspecified `**kwargs`. It's important that different methods share the same API - whatever we use for `cf.wi` should be the same for `cf.Query`.

Reply:
Fair enough. Happy to change accordingly. Will wait until I get to the office so we can talk about the API in-person, though, before making any changes.